test/coverlet.core.tests/Coverage/InstrumenterHelper.cs

```diff
@@ -484,7 +494,7 @@ class Logger : ILogger
     {
         string _logFile;
 
-        public Logger(string logFile) => _logFile = logFile;
+        public Logger(string logFile = "") => _logFile = logFile;
```

Comment:
why default `"`?

Reply:
Because for me this was the easiest way to let the container resolve this ```Ilogger``` instance. I now removed the default from the constructor and added the parameter when adding the ```Ilogger``` instance to the container.

Reply:
Ok, there is a problem, this Logger now is used during creation of `IInstrumentationHelper` and it will be used only in case of `BadImageFormatException`.
But we do not expect that exception on testing, so my proposal is to pass a `Mock<ILogger>()` that in case of `LogWarning` it will throw exception similar to `Unexpected warning message "xxx" during testing`, because otherwise it will throw invalid file name for `""`.
So I would revert this and add a mock, what do you think?

Reply:
Yes that's better. Done.